### DISCUSSION TITLE
Explanation why default in Credential

### DISCUSSION THREAD

**Discussion by sass1997** [vmware-tanzu/velero#7040] [Community support Q&A] [answered]
I would like to know why default is in the credential and what do I need to change in the backup storage location if I'm not using default.

[default] <----
aws_access_key_id=<AWS_ACCESS_KEY_ID>
aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>

**Comment by mateusoliveira43** [comment, marked as answer]
This is AWS syntax, not Velero. Reference https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-format

Did not find in Velero docs (this can be a issue), but I assume in install command you would pass `profile=[your profile name]` to `--backup-location-config` flag. Example https://velero.io/docs/v1.12/contributions/oracle-config/#install-velero

**Reply by mateusoliveira43** [reply]
Just to add the docs for `--backup-location-config` flag. It is mentioned in Velero docs, but lives in `velero-plugin-for-aws` repo https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/main/backupstoragelocation.md
